### PR TITLE
integrate questions api to front end

### PR DESCRIPTION
**Pull request by chuakid**

### CODE REVIEW

File: frontend/vite.config.ts [hunk 1]
```diff
@@ -1,7 +1,16 @@
...
+import react from "@vitejs/plugin-react";
 
 // https://vitejs.dev/config/
 export default defineConfig({
   plugins: [react()],
-})
+  server: {
+    proxy: {
+      "/api/v1/questions": {
+        target: "http://localhost:8080",
```
Comment (kxrt) [inline frontend/vite.config.ts:10]: Might be more optimal to use `http://question-service:8080` and `http://mongo-express:8081` to keep connections within the Docker network

**Review by kxrt — APPROVED** [review]
LGTM!

File: frontend/vite.config.ts (hunk 1 above)
Comment (chuakid) [reply frontend/vite.config.ts:10]: Can't do that this is the front end

Comment (chuakid) [reply frontend/vite.config.ts:10]: Unless you build it into a docker container but at that point you'd already build it so the proxy wouldn't be there

Comment (kxrt) [reply frontend/vite.config.ts:10]: Ah that's fair I was talking along the lines of when we integrate this into an overarching docker-compose file for Assignment 4

**Pull request merged**